Modules/CalcPerform.lua

```diff
@@ -805,6 +805,9 @@ function calcs.perform(env)
 			local globalCooldown = modDB:Sum("BASE", nil, "GlobalWarcryCooldown")
 			local globalCount = modDB:Sum("BASE", nil, "GlobalWarcryCount")
 			local uptime = m_min(full_duration / actual_cooldown, 1)
+			if modDB:Flag(nil, "Condition:WarcryMaxHit") then
```

Comment:
This needs to be moved below the modDB:Flag() check on line 813 or it will be overwritten when the Notable "Call to Arms" is taken and thus "WarcryShareCooldown" flag is true.

Reply:
Whoops, good catch! Fixed.